src/app_service/service/community/service.nim

```diff
@@ -635,11 +639,48 @@ QtObject:
   proc importCommunity*(self: Service, communityKey: string) =
     try:
       let response = status_go.importCommunity(communityKey)
```

Comment:
Is this call blocking the GUI thread?

Reply:
@osmaczko yes it is, we discussed that in private chat already. So the only way to avoid this is to call this in an async way. I just used the way how it was in the "old" master, but will try to update it now and make an async call to the status import community end point.

Reply:
I triggered this discussion on purpose. We should decide how to not block GUI thread.

According to our private chat discussion, the ideal solution would be to make status_go API async by nature. That is `importCommunity` should not block, it should be treated just as a simple command. In order to get the response, the App should be connected to `communityImported` signal and react accordingly:
```
proc importCommunity*(self: Service, communityKey: string) =
    status_go.importCommunity(communityKey)
    setCommunityIsBeingImported(communityKey) # GUI shows busy rotator/popup
...

self.events.on(SignalType.CommunityImported.event) do(e: Args):
    handleImportedCommunity(e)
...
```
Note that this approach would require changes in status_go.

Another solution would be to execute status_go calls in a separate thread:
```
proc importCommunity*(self: Service, communityKey: string) =
    status_go_wrapper.importCommunity(communityKey).then(handleImportedCommunity(response))
    setCommunityIsBeingImported(communityKey) # GUI shows busy rotator/popup
```

Reply:
@osmaczko yes, that's what we discussed.